Cargo.toml

```diff
@@ -44,13 +46,13 @@ mach2 = "0.4"
 [dev-dependencies]
 # Minidump-processor is async so we need an executor
 futures = { version = "0.3", features = ["executor"] }
-minidump = "0.16.0"
-memmap2 = "0.5"
+minidump = "0.17.0"
+memmap2 = "0.7"
 
 [target.'cfg(target_os = "macos")'.dev-dependencies]
 # We dump symbols for the `test` executable so that we can validate that minidumps
 # created by this crate can be processed by minidump-processor
 dump_syms = { version = "2.0.0", default-features = false }
-minidump-processor = { version = "0.16.0", default-features = false }
+minidump-processor = { version = "0.17.0", default-features = false }
```

Comment:
We need to add `minidump-unwind` as a dependency as some of the functionality we're using in the tests has been split out into that crate.